src/lib.rs

```diff
@@ -14,10 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+//! Virtio socket support for Rust.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-
-//! Virtio socket support for Rust.
+#[deny(warnings)]
```

Comment:
Should probably be `#![deny(warnings)]`?

Reply:
Good catch!